tests/foreman/api/test_smartproxy.py

```diff
@@ -238,14 +240,15 @@ def test_positive_import_puppet_classes(self):
             proxy = entities.SmartProxy(url=url).create()
             result = proxy.import_puppetclasses()
             self.assertEqual(
-                result.message,
+                result['message'],
```

Comment:
I've accidentally discovered this, `result` is a dict, not an object, so it was causing attributeError.